sdk/mosaic.go

```diff
@@ -152,30 +185,37 @@ func (dto mosaicPropertiesDTO) toStruct() *MosaicProperties {
 	)
 }
 
-func (ref *mosaicInfoDTO) setMosaicInfo() (*MosaicInfo, error) {
-	publicAcc, err := NewAccountFromPublicKey(ref.Mosaic.Owner, NetworkType(1))
+type mosaicContext struct {
+	networkType NetworkType
+}
+
+func (ref *mosaicInfoDTO) setMosaicInfo(context *mosaicContext) (*MosaicInfo, error) {
```

Comment:
Please rename this method to toStruct(). All DTO structures have method toStruct() which returns normal object

Reply:
toMosaicInfo is better name